Adyen.Test/HeaderRequestTest.cs

```diff
@@ -30,18 +30,17 @@ namespace Adyen.Test
     [TestClass]
     public class HeaderRequestTest
     {
-
-        private readonly HttpUrlConnectionClient _httpUrlConnectionClient;
-        private readonly string _endpoint = "https://endpoint:8080";
+        private const string _endpoint = "https://endpoint:8080";
 
         public HeaderRequestTest()
         {
```

Comment:
If this constructor is not used we can remove it

Reply:
Good point. Will remove

Reply:
Removed